configure.ac

```diff
@@ -66,6 +66,16 @@ AC_ARG_ENABLE([extratests],
 	AS_HELP_STRING([--enable-extratests], [Enable extra tests that may be unreliable on some platforms]))
 AM_CONDITIONAL([ENABLE_EXTRATESTS], [test "x$enable_extratests" = xyes])
 
+AC_ARG_ENABLE([tests],
+       [AS_HELP_STRING([--disable-tests], [Enable tests (default: yes)])],
```

Comment:
If using "--disable-tests", help message would be "Disable tests" ?
IMHO, default value description is not needed like "--disable-asm".